src/Pages/SingleVideoPage/SingleVidePageComponents/RelatedVideos/RelatedVideos.jsx

```diff
@@ -0,0 +1,17 @@
+import { VideoCard } from "../../../../Components";
```

Comment:
Suggestion : Try using absolute imports instead of relative imports. This will make importing and managing of code during refactoring easier.

Reference : https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

Reply:
I dont know much about absolute imports. Thanks for providing the link i will read about it for sure!